common/features/org.jboss.tools.common.all.feature/pom.xml

```diff
@@ -8,5 +8,6 @@
 	</parent>
 	<groupId>org.jboss.tools.common.features</groupId>
 	<artifactId>org.jboss.tools.common.all.feature</artifactId> 
+	<version>3.8.4-SNAPSHOT</version>
```

Comment:
Not true. This is how we differentiate a specific plugin from the root pom's version -- the alternative is to upversion everything in the project, which results in way more changes than are needed. Parent pom has a completely different version - 4.4.3.AM1-SNAPSHOT, to be specific.